.licenserc.yaml

```diff
@@ -1,13 +1,13 @@
-#
-# SPDX-FileCopyrightText: 2025 Lifely
+# SPDX-FileCopyrightText: 2025 INFO.nl
 # SPDX-License-Identifier: EUPL-1.2+
+
 #
 # Configuration file for https://github.com/apache/skywalking-eyes
 # to perform checks (and optionally fixes) on SPDX copyright headers in source files
 header:
   license:
     content: |      
-       SPDX-FileCopyrightText: 2025 Lifely
+       SPDX-FileCopyrightText: 2025 INFO.nl
```

Comment:
We should not change this in this PR. That means you will need to changed every single file in our code base. That should be a separate PR and JIRA task.

Also, maybe we should not change it to INFO.nl. We should have a discussion about this first!

Reply:
This only applies to new files without headers, but I reverted it back to Lifely